packages/frontend/src/components/common/PivotTable/TitleCell.tsx

```diff
@@ -1,34 +1,47 @@
-import { TitleFieldValue } from '@lightdash/common';
-import { Box } from '@mantine/core';
+import { Box, Tooltip } from '@mantine/core';
 import { FC } from 'react';
 
 interface TitleCellProps {
-    title: TitleFieldValue;
+    className?: string;
+    description?: string;
     level?: number;
+    isEmpty: boolean;
+    isHeaderTitle: boolean;
 }
 
-const TitleCell: FC<TitleCellProps> = ({ children, title, level = 1 }) => {
-    const isEmpty = !title?.fieldId;
-    const isHeaderTitle = title?.titleDirection === 'header';
-
+const TitleCell: FC<TitleCellProps> = ({
+    children,
+    className,
+    description,
+    level = 1,
+    isEmpty,
+    isHeaderTitle,
+}) => {
     return (
-        <Box
-            component="th"
-            ta="right"
-            style={{
-                textAlign: isHeaderTitle ? 'right' : undefined,
-            }}
-            sx={(theme) => ({
-                fontWeight: 600,
-                backgroundColor: isEmpty
-                    ? theme.white
-                    : isHeaderTitle
-                    ? theme.colors.gray[level - 1]
-                    : theme.colors.gray[0],
-            })}
+        <Tooltip
+            withArrow
+            withinPortal
+            multiline
+            disabled={!description}
+            label={description}
         >
-            {children}
-        </Box>
+            <Box
+                component="th"
+                className={className}
+                style={{
+                    textAlign: isHeaderTitle ? 'right' : undefined,
+                }}
+                sx={(theme) => ({
+                    backgroundColor: isEmpty
+                        ? theme.white
+                        : isHeaderTitle
+                        ? theme.colors.gray[level]
```

Comment:
Was the level intended to change from `level-1` -> `level` ?

Reply:
yes! to make highlighting on hover more obvious